common/src/test/java/com/datastrato/gravitino/config/TestConfigEntryList.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.config;
+
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class TestConfigEntryList {
+
+  private final ConcurrentMap<String, String> configMap = new ConcurrentHashMap<>();
+
+  @BeforeEach
+  public void initializeConfigMap() {
+    configMap.put("gravitino.test.string.list", "test-string-1,test-string-2,test-string-3");
+  }
+
+  @AfterEach
+  public void clearConfigMap() {
+    configMap.clear();
+  }
+
+  @Test
+  public void testConfWithDefaultValue() {
+    ConfigEntry<List<String>> testConf1 =
+        new ConfigBuilder("gravitino.test.list")
+            .doc("test")
+            .internal()
+            .stringConf()
+            .checkValue(value -> value == "test-string-1,test-string-2,test-string-3", "error")
```

Comment:
Maybe we should call method `readFrom` for `testConf1`. Because the method `checkValue` is called when we call the method `readFrom`.

Reply:
I see, sure! thx for heads-up~